Compiler/src/main/java/com/adguard/compiler/SettingUtils.java

```diff
@@ -33,35 +31,6 @@
  */
 public class SettingUtils {
 
-    private final static String LOCAL_SCRIPT_RULES_FILE_TEMPLATE = "/**\r\n" +

```

Comment:
I don't like that we got rid of this description (starting from `By the rules of`).

I realize, that we can't put it as a comment into the local-scripts-file.
However, we can make it a field:
```
{
    comment: "Here goes the description",
    rules: [
       ...
    ]
}
```